types/database/database.d.ts

```diff
@@ -227,6 +227,11 @@ export type HandleGroupMembershipForMemberArgs = PrepareOnly & {
   groups?: Group[];
 }
 
+export type HandleGroupTeamsForTeamArgs = PrepareOnly & {
+  teamId: string;
+  groups?: Group[];
```

Comment:
`groups?: Group[];` should this be `groupTeams: GroupTeam[]` ?

Reply:
No - it expects `Group[]` and transforms it in the handler to `GroupTeam[]`